api.swagger.yaml

```diff
@@ -1355,27 +1355,108 @@ paths:
           name: fulltext_search
           schema:
             type: string
-        - description: Filter by UUID of a category.
+        - description: 'Filter by UUID of a category. Multiple values may be supplied.'
           in: query
           name: category
           schema:
-            type: string
+            type: array
+            items:
+              type: string
+              format: uuid
         - description: Filter by groups the current user is a member of.
           in: query
           name: my_groups
           schema:
             type: boolean
             default: false
+        - description: 'Filter groups that are administered by the user UUID. Multiple values may be supplied.'
+          in: query
+          name: admin_uuid
+          schema:
+            type: array
+            items:
+              type: string
+              format: uuid
+        - description: 'Filter groups that where created after (greater than or equal) the timestamp (in seconds).'
+          in: query
+          name: created_start_timestamp
+          schema:
+            type: integer
+            example: 1563357496
+        - description: 'Filter groups that where created before (less than or equal) the timestamp (in seconds).'
+          in: query
+          name: created_end_timestamp
+          schema:
+            type: integer
+            example: 1563357496
+        - description: 'Filter groups with last posts after (greater than or equal) the timestamp (in seconds).'
+          in: query
+          name: last_post_start_timestamp
+          schema:
+            type: integer
+            example: 1563357496
+        - description: 'Filter groups with last posts before (less than or equal) the timestamp (in seconds).'
+          in: query
+          name: last_post_end_timestamp
+          schema:
+            type: integer
+            example: 1563357496
+        - description: 'Filter groups on the privacy setting. Multiple values may be supplied.'
+          in: query
+          name: privacy_setting
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - Public
+                - Private
+                - Secret
+        - description: 'Filter groups with the membership types. Multiple values may be supplied.'
+          in: query
+          name: members_type
```

Comment:
let's take this out for now, I don't think it's needed

Reply:
done